cumulusci/tasks/salesforce/tests/test_EnsureRecordTypes.py

```diff
@@ -253,6 +253,33 @@ def test_no_action_if_existing_record_types(self):
         self.assertFalse(task.options["generate_business_process"])
         self.assertFalse(task.options["generate_record_type"])
 
+    def test_second_rt_if_force_create(self):
+        # Asserts a second Record Type is added even when RTs already exist when force_create is True
+        task = create_task(
+            EnsureRecordTypes,
+            {
+                "record_type_developer_name": "NPSP_Default",
+                "record_type_label": "NPSP Default",
+                "record_type_description": "Default Account Record Type created by NPSP.",
+                "sobject": "Account",
+                "force_create": True,
+            },
+        )
+
+        task.sf = mock.Mock()
+        task.sf.Account = mock.Mock()
+        task.sf.Account.describe = mock.Mock(return_value=OPPORTUNITY_DESCRIBE_WITH_RTS)
+        task._infer_requirements()
+
+        with temporary_dir():
+            task._build_package()
+            with open(os.path.join("objects", "Account.object"), "r") as f:
+                opp_contents = f.read()
```

Comment:
This variable name may be confusing. `obj_contents`?

Reply:
This way of reading the file is fine, but it's also possible to do it in a single line using pathlib:

    obj_contents = pathlib.Path("objects", "Account.object").read_text()

Also, _probably_ doesn't matter here, but it's good to get in the habit of always specifying `encoding="utf-8"` when opening a file or using read_text. There is often a different default encoding on Windows.

Reply:
discussed with DR... we will put off a pathlib refactor until we do the entire file.